packages/sanity/src/core/form/inputs/PortableText/text/textStyles.tsx

```diff
@@ -10,6 +10,12 @@ export const TextContainer = styled.div`
   display: block;
 `
 
+/**
+ * Component for wrapping children inside a user's custom style component.
+ * This ensures that we have a TextContainer there (see comment above).
+ */
+export const Custom = ({children}: TextStyleProps) => <TextContainer>{children}</TextContainer>
```

Comment:
Since `TextContainer` is already exported – could we remove this one and instead use `TextContainer` in `Style.tsx`?

Reply:
I was giving this some thoughts yesterday and I wouldnt t leak that logic with the android issue out other places. Conceptually it is easier to deal with various styles. Internally we use Custom when user style.